governator-test-junit/build.gradle

```diff
@@ -1,6 +1,11 @@
 apply plugin: 'java'
 
 dependencies {
-    compile     project(':governator')
-    compile     "junit:junit:4.11"
+    compile    project(':governator')
+    compile    project(':governator-api')
+    compile    project(':governator-core')
+    compile    'com.netflix.archaius:archaius2-guice:2.0.6'
+    compile    'com.netflix.archaius:archaius2-test:2.0.6'
+    compile    'junit:junit:4.12'
```

Comment:
I thought all the tests in governator-* moved to JUnit from TestNG, and that in the future they might be using your annotations for unit testing.  I think there needs to be a global JUnit version, and it should be the same here as it is in 'testCompile'  configuration elsewere in the project family.  Otherwise we could end up fail our own tests for JUnit version drift.